src/node.ts

```diff
@@ -8,6 +8,8 @@ class Node extends VEventTarget {
   constructor(name: string) {
     super();
     this.name = name || 'Untitled';
+    // this.dispatchEvent(new VEvent('nodeCreated'));
+    // The dispatchEvent is commented, becouse the event will always fire before added handlers.
```

Comment:
This is true. But perhaps the controller should not care about node creation, unless it is added to the nework. So, an "addNode" event can be triggered when the node is added.